geometry_msgs/PoseStamped target_pose
---
std_msgs/String result
